server.js

```diff
@@ -23,6 +23,17 @@ app.set('port', (process.env.PORT || 3000));
 app.use('/', express.static(path.join(__dirname, 'public')));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({extended: true}));
+app.use(function(req, res, next) {
+    //set permissive CORS headers
+    res.setHeader('Access-Control-Allow-Origin', '*');
+    res.setHeader('Access-Control-Allow-Methods', 'GET,OPTIONS,HEAD,PUT,POST,DELETE,PATCH');
```

Comment:
I agree, except I believe that the preflight is also needed for POST? so I could remove the method and headers lines.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS